independent-publisher-connectors/Revue/apiDefinition.swagger.json

```diff
@@ -0,0 +1,676 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Revue",
+    "description": "Revue makes it easy for writers and publishers to send editorial newsletters \u2014 and get paid.",
```

Comment:
@Laskewitz Unicodes - The connector's swagger still have unicodes which may not be supported in all the Power Automate page, Can you please replace all the unicodes with respective characters in the English language.

Reply:
@sriyen-msft Fixed!